### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by Sanjit-Prasad**
Need some help in uploading files via New.js

### CODE REVIEW

File: app/assets/javascripts/pages/Error404.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+              <meta name="viewport" content="width=device-width,initial-scale=1">
+              <style>
+              .rails-default-error-page {
+                background-color: #EFEFEF;
+                color: #2E2F30;
+                text-align: center;
+                font-family: arial, sans-serif;
+                margin: 0;
+              }
+
```
Comment (akshitabansal97) [inline app/assets/javascripts/pages/Error404.js:17]: You should refrain to add css styling in the same same, instead make a separate file for css and then include it

File: app/assets/javascripts/pages/Show.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+function myFunction(id) {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+      document.getElementById("demo").innerHTML = "Interview successfully deleted.";
+    }
+  };
+  xhttp.open("DELETE", "/interviews/" + id, true);
+  xhttp.send();
```
Comment (akshitabansal97) [inline app/assets/javascripts/pages/Show.js:26]: Why Xml format is used here? Any specific reason?

File: app/assets/javascripts/pages/Error404.js (hunk 1 above)
Comment (Sanjit-Prasad) [reply app/assets/javascripts/pages/Error404.js:17]: will do that.

File: app/assets/javascripts/pages/Show.js (hunk 2 above)
Comment (Sanjit-Prasad) [reply app/assets/javascripts/pages/Show.js:26]: For calling delete method from controller.

**Pull request merged**